.github/CODEOWNERS

```
@@ -0,0 +1 @@
+* @bajtos @juliangruber @pyropy @NikolasHaimerl
```

I'd like to delegate the ownership to others.

```suggestion
* @juliangruber @pyropy @NikolasHaimerl
```